src/bindings/config.ts

```diff
@@ -25,6 +25,10 @@ export const loadConfig = async (): Promise<BotConfig> => {
       token: process.env.TELEGRAM_BOT_TOKEN ?? "",
       delay: process.env.TELEGRAM_BOT_DELAY ? Number(process.env.TELEGRAM_BOT_DELAY) : DEFAULT_BOT_DELAY,
     },
+    git: {
+      org: process.env.ORG_NAME ?? "ubiquity",
+      repo: process.env.REPO_NAME ?? "ubiquity-dollar",
```

Comment:
We should log a warning if it goes to the fallback, our repo. It can cause confusion for partners setting this up if they forgot to set it to their repo.

Reply:
added fallback warning